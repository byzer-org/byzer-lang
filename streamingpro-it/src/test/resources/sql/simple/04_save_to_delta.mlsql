save overwrite t2 as delta.`public.t1`;
load delta.`public.t1` as t3;
select * from t3 limit 5 as output;