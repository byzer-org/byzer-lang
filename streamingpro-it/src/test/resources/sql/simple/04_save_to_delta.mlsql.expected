owner,join_date,leave_date2
Unknown,1970-01-01,2099-01-01
Owner0,2018-01-01,2099-01-01
Owner156f6b6e,2018-01-02,2019-12-22
Owner16a91f7d,2018-01-03,2099-01-01
Owner18c75786,2018-01-04,2099-01-01